sql/core/src/test/scala/org/apache/spark/sql/CachedTableSuite.scala

```diff
@@ -64,6 +66,13 @@ class CachedTableSuite extends QueryTest with SQLTestUtils with SharedSQLContext
     maybeBlock.nonEmpty
   }
 
+  def isExpectStorageLevel(rddId: Int, level: DataReadMethod.DataReadMethod): Boolean = {
+    val maybeBlock = sparkContext.env.blockManager.get(RDDBlockId(rddId, 0))
```

Comment:
shall we also check that the block is present? If nothing is cached this would (wrongly IMHO) return true. I see that since you are checking `isMaterialized` this doesn't happen in your test cases, but this function may be reused in the future so I think it'd be better to have it behaving consistently to its name. Do you agree?